fire/io/bernese/bernese.py

```diff
@@ -0,0 +1,369 @@
+        første_sektion_begyndelse = (
+            addneq_data.index(
+                " Sol Station name         Typ Correction  Estimated value  RMS error   A priori value Unit    From                To                  MJD           Num Abb  \n"
+            )
+            + 2
+        )
+        # Vi kan desværre ikke antage rækkefølge på sektionerne, men denne sektion ser ud til at være færdig når den
+        # efterfølges af 1-2 tomme linjer
+        første_sektion_ende = addneq_data.index("\n", første_sektion_begyndelse)
+        første_sektion = addneq_data[første_sektion_begyndelse:første_sektion_ende]
+
+        # Så skal der udtrækkes observationslængder fra den første udklippede sektion
+        for linje in første_sektion:
+            if not linje.isspace():  # skip evt. tomme linjer
+                observation = addneq_parse_observationline(linje)
+                self[observation["STATION NAME"]].obsstart = datetime.strptime(
+                    observation["FROM"], "%Y-%m-%d %H:%M:%S"
+                )
+                self[observation["STATION NAME"]].obsslut = datetime.strptime(
+                    observation["TO"], "%Y-%m-%d %H:%M:%S"
+                )
+                self[observation["STATION NAME"]].rms_fejl = observation["RMS_ERROR"]
+
+        # Det samme gør sig ikke gældende med afsnittet om spredninger - der er som regel en tom linje efter hver
+        # station, men der synes overskriften altid at være den samme
+        # Dog er denne sektion til tider slet ikke til stede - i så fald skipper vi den
+        try:
+            anden_sektion_begynd = (
+                addneq_data.index(" Comparison of individual solutions:\n") + 3
+            )
+            anden_sektion_ende = addneq_data.index(
+                " Variance-covariance scaling factors:\n"
+            )
+            anden_sektion = addneq_data[anden_sektion_begynd:anden_sektion_ende]
+
+            # Og spredning fra den anden udklippede sektion
+            stations_rms = {}
+            for station in self:
+                stations_rms[station] = {}  # opret en tabel med navne fra CRD parsing
+            for linje in anden_sektion:
+                if not linje.isspace():  # skip evt. tomme linjer
+                    spredningsset = addneq_parse_spreadline(linje)
+                    stations_rms[spredningsset["STATION NAME"]][
+                        spredningsset["DIRECTION"]
+                    ] = spredningsset["SPREAD"]
+            for station in stations_rms:
+                if stations_rms[station] == {}:  # spring over stationer uden værdier
+                    continue
+                n = stations_rms[station]["N"]
+                e = stations_rms[station]["E"]
+                u = stations_rms[station]["U"]
+                rms = RMS(n=n, e=e, u=u)
+                self[station].spredning = rms
+        except ValueError:
+            print("Ingen spredningssektion i ADDNEQ-fil - skipper...")
```

Comment:
Det er ikke så fedt at skrive til STDOUT med bibliotekskode som dette. Jeg er enig i at det er smart at gøre opmærksom på at vi har med en fil med begrænset indhold at gøre. Ideelt set havde vi en logger det kunne spyttes ud til men det har vi ikke i øjeblikket. Mit forslag er at udlade advarslen for nu. Heldigvis kan man jo konstatetere at noget mangler og reagerer på det fra det kaldende kode.
```suggestion
        except ValueError:
            pass
```

Reply:
Fikset i seneste commits - men jo måske burde vi have en logger!

Reply:
Det skal vi helt sikkert have. Jeg bør lave et issue på det. Det vil gøre mange ting lettere